AFQ/data.py

```diff
@@ -940,25 +980,145 @@ def read_mni_template(resolution=1, mask=True):
         Whether to mask the data with a brain-mask before returning the image.
         Default : True
 
+    weight: number or str, optional
+        Which relaxation technique to use.
+        Should be either 1 or 2, correponding to T1 or T2.
+        Default : 2
+
     Returns
     -------
-    nib.Nifti1Image class instance containing masked or unmasked T2 template.
+    nib.Nifti1Image class instance
+    containing masked or unmasked T1w or template.
 
     """
     template_img = nib.load(str(tflow.get('MNI152NLin2009cAsym',
                                           desc=None,
                                           resolution=resolution,
-                                          suffix='T2w',
+                                          suffix=f'T{weight}w',
                                           extension='nii.gz')))
     if not mask:
         return template_img
     else:
-        mask_img = nib.load(str(tflow.get('MNI152NLin2009cAsym',
-                                          resolution=resolution,
-                                          desc='brain',
-                                          suffix='mask')))
+        return _apply_mask(template_img, resolution)
 
-        template_data = template_img.get_fdata()
-        mask_data = mask_img.get_fdata()
-        out_data = template_data * mask_data
-        return nib.Nifti1Image(out_data, template_img.affine)
+
+fetch_biobank_templates = \
+    _make_fetcher(
+        "fetch_biobank_templates",
+        op.join(afq_home,
+                'biobank_templates'),
+        "http://biobank.ctsu.ox.ac.uk/showcase/showcase/docs/",
+        ["bmri_group_means.zip"],
+        ["bmri_group_means.zip"],
+        data_size="1.1 GB",
+        doc="Download UK Biobank templates",
+        unzip=True)
+
+
+def read_fa_template(mask=True):
```

Comment:
Let's clearly denote that this is from the UKBB:

```suggestion
def read_ukbb_fa_template(mask=True):
```